src/interceptors/server/conversionInterceptorFactory.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Server-side JSON -> protobuf -> JSON handler wrapper (factory)
+ *
+ * @param {jsonToProtobuf} jsonToProtobuf
+ * @param {protobufToJson} protobufToJson
+ * @param {function(grpc.Call, function(Error|null, jspb.Message|null))} rpcMethod
+ *
+ * @returns {wrappedMethodHandler}
+ */
+function conversionInterceptorFactory(jsonToProtobuf, protobufToJson, rpcMethod) {
```

Comment:
Actually, it’s not interceptor right? It might confuse because server interceptor in the spec (which is not implemented yet) looks different. I would move it to DAPI, since all GRPS related infrastructure logic is there (i.e. error handler). We will move it to separate repo when migrate Drive to GRPC.